README.md

```diff
@@ -117,14 +117,14 @@ export {MyType}
 
 ## Utils [![Build Status](https://travis-ci.org/rpominov/static-land.svg?branch=master)](https://travis-ci.org/rpominov/static-land) [![Coverage Status](https://coveralls.io/repos/github/rpominov/static-land/badge.svg?branch=master)](https://coveralls.io/github/rpominov/static-land?branch=master)
 
-We also have `static-land` package on NPM that provides some useful utilities (not many at the moment).
+We also have `static-land` package on npm that provides some useful utilities (not many at the moment).
 
 * [API reference](docs/API.md)
 
 ### Installation
 
-```sh
-npm install static-land
+```console
+$ npm install static-land
```

Comment:
I heard that `$` confuses people, they may not know what it means and try to paste it to terminal. Let's leave it without `$`.